pkg/cloudprovider/aws/aws.go

```diff
@@ -1294,8 +1295,8 @@ func (s *AWSCloud) describeLoadBalancer(region, name string) (*elb.LoadBalancerD
 
 	response, err := elbClient.DescribeLoadBalancers(request)
 	if err != nil {
-		if awsError := aws.Error(err); awsError != nil {
-			if awsError.Code == "LoadBalancerNotFound" {
+		if awsError := err.(awserr.Error); awsError != nil {
```

Comment:
Thanks for catching this! I'll update the code based on your suggestion. This also appears to be the same guard they provide as an example.

https://github.com/aws/aws-sdk-go/blob/master/aws/awserr/error.go#L15